src/pages/api/chooseValidity.ts

```diff
@@ -16,28 +14,57 @@ export const isInvalidValidityNumber = (req: NextApiRequestWithSession): boolean
         return true;
     }
 
-    if (validityInput > 366 || validityInput < 1) {
+    if (validityInput > 1000 || validityInput < 1) {
         return true;
     }
 
     return false;
 };
 
-export const setSession = (req: NextApiRequestWithSession, res: NextApiResponse, error = ''): void => {
-    const daysValid = req.body.validityInput;
+export const validityInputErrorCheck = (errorInfo: ErrorInfo[], validityInput: string): ErrorInfo[] => {
```

Comment:
With the below error messages, should we be telling the user what they **_should_** enter, rather than just what they **_shouldn't_**?

Reply:
Yeah I could check all 3 conditions at once and tell them it has to be a whole positive number etc. True.